src/components/App/App.jsx

```diff
@@ -123,6 +129,7 @@ class App extends React.Component {
 
   finishedWithAuth(user) {
     this.setState({ user })
+    this.onUserLoad(user.login)
```

Comment:
Lines 131-132 can be reduced to a single line if you make `onUserLoad` expect a full user object and have it do the setState call.